### PR TITLE
[pt-br] Add docs/tasks/configure-pod-container/configure-projected-volume-storage.md

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Configurando um Pod Para Usar um Volume Projetado Para Armazenamento
+content_type: task
+weight: 70
+update_date: 2022-07-18
+origin_version: 1.24
+contributors: DonatoHorn
```
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:7]: Git tracks the date of changes and who made them. These fields don't look necessary.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+{{< glossary_tooltip text="Secrets" term_id="secret" >}} a partir de arquivos locais. 
+Você então cria um Pod que executa um contêiner, usando um volume
+[`projetado`](/docs/concepts/storage/volumes/#projected) 
+para montar as `Secrets` dentro do mesmo diretório compartilhado.
+
+Aqui está o arquivo de configuração para o Pod:
+
+{{< codenew file="pods/storage/projected.yaml" >}}
+
+1. Crie os Segredos:
```
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:42]: If writing the translated term “Segredo”, don't capitalize the first letter.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+1. No seu shell, verifique se o diretótio `projected-volume` 
+contém suas fontes projetadas:
+
+    ```shell
+    ls /projected-volume/
+    ```
+
+## Limpeza
+
+Delete o Pod e o `Secrets`:
```
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:89]:
```suggestion
Delete o Pod e o Secret:
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+kubectl delete pod test-projected-volume
+kubectl delete secret user pass
+```
+
+
+
+## {{% heading "whatsnext" %}}
+
+* Aprensa mais sobre volume [`projetado`](/docs/concepts/storage/volumes/#projected).
+* Leia o documento de desenho de [volume tudo-em-um](https://git.k8s.io/design-proposals-archive/node/all-in-one-volume.md) design document.
```
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:101]: I would localize “design document” by translating it.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+origin_version: 1.24
+contributors: DonatoHorn
+reviewers:
+- jpeeler
+- pmorie
+---
+
+<!-- overview -->
+Esta página mostra como usar um Volume
+[`projetado`](/docs/concepts/storage/volumes/#projected) para montar diversas
```
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:15]:
```suggestion
[`projected`](/docs/concepts/storage/volumes/#projected) para montar diversas 
```
The Kubernetes API does not recognize the value `projetado`.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 1 above)
Comment (DonatoHorn) [reply content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:7]: Hi, I'm new contributor on pt-br docs, so my difficults are identify pages (original version) than where used base-line for internacionalizations, in pt-br docs we have pages from v1.18, 1,19, 1.20, 1.21, 1.22, 1.23, 1.24, how can I guess what pages need update. I'ts nice to do this, make a diff in the english docs, so the pages that need update will be shown. But in pt-br we don't have versioning, so if I have a origin_version  in the docs I can use a grep to show what a version used in, and assign the contributor to ask if interested in update the docs they translated.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 4 above)
Comment (DonatoHorn) [reply content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:101]: Sorry, I Just forgot to erase the end of the sentence. Done.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 1 above)
Comment (sftim) [reply content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:7]: That might be a nice enhancement to make. However, before you do this, we should get consensus between the localization teams to adopt that approach.

I'm afraid that changing this unilaterally is likely to be overall unhelpful.

**Comment by DonatoHorn** [comment]
Thank you for reviewing.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md (hunk 1 above)
Comment (DonatoHorn) [reply content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:7]: I don't  know who I need to talk about.

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/configure-pod-container/configure-projected-volume-storage.md

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
+---
+title: Configurando um Pod Para Usar um Volume Projetado Para Armazenamento
+content_type: task
+weight: 70
+reviewers:
+- jpeeler
+- pmorie
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:7]: This section can be removed.

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+---
+
+<!-- overview -->
+Esta página mostra como usar um Volume
+[`projected`](/docs/concepts/storage/volumes/#projected) para montar diversas 
+fontes de volumes existententes no mesmo diretório. Atualmente, volumes `secret`, 
+`configMap`, `downwardAPI`, e `serviceAccountToken` podem ser projetados.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:14]:
```suggestion
Esta página mostra como usar um Volume
[projetado](/pt-br/docs/concepts/storage/volumes/#projetado) para montar diversas 
fontes de volumes existentes no mesmo diretório. Atualmente, volumes `secret`, 
`configMap`, `downwardAPI`, e `serviceAccountToken` podem ser projetados.
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+{{< /note >}}
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+<!-- steps -->
+## Configure um volume projetado para um pod
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:27]:
```suggestion
## Configure um volume projetado em um Pod
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+Neste exercício, você cria o nome de usuário e senha 
+{{< glossary_tooltip text="Secrets" term_id="secret" >}} a partir de arquivos locais. 
+Você então cria um Pod que executa um contêiner, usando um volume
+[`projected`](/docs/concepts/storage/volumes/#projected) 
+para montar as `Secrets` dentro do mesmo diretório compartilhado.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:33]:
```suggestion
Neste exercício, você cria {{< glossary_tooltip text="Secrets" term_id="secret" >}} com nome de usuário e senha a partir de arquivos locais. 
Você então cria um Pod que executa um contêiner, usando um volume
[projetado](/pt-br/docs/concepts/storage/volumes/#projetado) 
para montar os objetos Secret dentro do mesmo diretório compartilhado.
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+Aqui está o arquivo de configuração para o Pod:
+
+{{< codenew file="pods/storage/projected.yaml" >}}
+
+1. Crie os segredos:
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:39]:
```suggestion
1. Crie os Secrets:
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+
+    ```shell
+    # Crie arquivos contendo o username e password:
+    echo -n "admin" > ./username.txt
+    echo -n "1f2d1e2e67df" > ./password.txt
+
+    # Empacote esses arquivos em `secrets`:
+    kubectl create secret generic user --from-file=./username.txt
+    kubectl create secret generic pass --from-file=./password.txt
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:48]:
```suggestion
    # Crie arquivos contendo o usuário e senha:
    echo -n "admin" > ./username.txt
    echo -n "1f2d1e2e67df" > ./password.txt

    # Empacote esses arquivos em objetos do tipo Secret:
    kubectl create secret generic user --from-file=./username.txt
    kubectl create secret generic pass --from-file=./password.txt
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Crie o Pod:
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/pods/storage/projected.yaml
+    ```
+
+1. Verifique se o contêiner do pod está em execução e, em seguida, 
+observe as alterações no pod:
+
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:59]:
```suggestion
1. Verifique que o contêiner do Pod está em execução e, em seguida, 
observe as alterações no Pod:

```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+    ```
+
+    A saída se parece com o seguinte:
+    ```
+    
+    NAME                    READY     STATUS    RESTARTS   AGE
+    test-projected-volume   1/1       Running   0          14s
+    ```
+
+1. Em outro terminal, pegue um shell para executar o contêiner:
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:71]:
```suggestion
1. Em outro terminal, inicie um shell para executar o contêiner:
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+    ```shell
+    kubectl exec -it test-projected-volume -- /bin/sh
+    ```
+
+1. No seu shell, verifique se o diretótio `projected-volume`
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:77]:
```suggestion
1. No seu shell, verifique se o diretório `projected-volume` 
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+```shell
+kubectl delete pod test-projected-volume
+kubectl delete secret user pass
+```
+
+
+
+## {{% heading "whatsnext" %}}
+
+* Aprensa mais sobre volume [`projected`](/docs/concepts/storage/volumes/#projected).
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:97]:
```suggestion
* Aprenda mais sobre volumes [projetados](/pt-br/docs/concepts/storage/volumes/#projetado).
```

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+* Leia o documento de desenho de [volume tudo-em-um](https://git.k8s.io/design-proposals-archive/node/all-in-one-volume.md).
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:98]:
```suggestion
* Leia o documento de design de [volume tudo-em-um](https://git.k8s.io/design-proposals-archive/node/all-in-one-volume.md).
```

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @DonatoHorn! 👋

Thank you very much for your work in this pull request. The content looks good, I just found a few minor changes needed for accuracy. If you disagree with any of the suggestions please feel free to reach out to me on the `#kubernetes-docs-pt` channel of Kubernetes Slack. 😃

---

Olá, @DonatoHorn! 👋

Muito obrigada pelo teu trabalho neste pull request. O conteúdo está bom, apenas alguns ajustes são necessários por motivo de precisão. Se discordares de qualquer das sugestões podes entrar em contato comigo pelo canal `#kubernetes-docs-pt` no Slack do Kubernetes. 😃

/label tide/merge-method-squash

**Comment by divya-mohan0209** [comment]
@stormqueen1990 : Any updates on reviewing the PR or can this be merged? It's been pending for ~3ish months.

**Comment by stormqueen1990** [comment]
> @stormqueen1990 : Any updates on reviewing the PR or can this be merged? It's been pending for ~3ish months.

Hi there, @divya-mohan0209! We'll take another look soon, thanks for flagging this!

/cc @MrErlison

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
+---
+title: Configurando um Pod Para Usar um Volume Projetado Para Armazenamento
+content_type: task
+weight: 70
+reviewers:
```
Comment (edsoncelio) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:5]: This section can be removed :)

File: content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+- jpeeler
+- pmorie
+---
+
+<!-- overview -->
+Esta página mostra como usar um Volume
+[projetado](/docs/concepts/storage/volumes/#projetado) para montar diversas
```
Comment (edsoncelio) [inline content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md:12]: You can use point to the pt-br link: pt-br/docs/concepts/storage/volumes/#projetado

**Comment by edsoncelio** [comment]
Just a few changes, after that will be LGTM :)

**Comment by edsoncelio** [comment]
Hey @DonatoHorn! 
Due to the lack of response, we will close this PR but feel free to reopen it If you want to make the changes requested.
/close

**Pull request closed without merging**